### ISSUE TITLE
Tags should only be focusable if they can be actioned (deleted / selected)

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#769]
### Expected Behaviour
There is no need to be able to tab to a Tag

### Actual Behaviour
Each tag group is tabable and then arrow keys are used to move between tags. When tags are a simple list they are not actionable elements and should not be in the tab order. When tags are removable then allow tabbing to the delete button directly.

#### Steps to Reproduce
https://opensource.adobe.com/spectrum-web-components/components/tags/examples
tab to the tag

#### Logs taken while reproducing problem

Note - lists/listitem are structural roles in ARIA not widget roles. If they really need to be tabable then they should be a layout grid not a list.

**Comment by Westbrook** [comment]
The Spectrum CSS and Spectrum Specification documentation for this seems like it is laking some clarity based on this extended context. @jnurthen do you have some time next week to touch base on this in-person to ensure that I fully understand the nuance here?

**Comment by jnurthen** [comment]
Sure. Please find a time in calendar. I'm sure there are probably a few more issues to discuss too.

**Comment by Westbrook** [comment]
Ensure focus of tag and tags when any subset of tag in tags in deletable / selectable.
